.github/workflows/zero-touch-copilot-merge.yml

```
@@ -0,0 +1,171 @@
+          }
+
+          BLOCKED=$(grep -E '^(Sources/Kumone/|Sources/KumoneLauncher/|ios/|Package\.swift$|Package\.resolved$|AppIcon\.icon/|\.github/workflows/|\.github/CODEOWNERS$)' <<<"$FILES" || true)
+          if [[ -n "$BLOCKED" ]]; then
+            gh pr edit "$PR_NUMBER" --repo "$REPO" --add-label "automation/blocked" >/dev/null
+            MARKER='<!-- zero-touch-protected-scope -->'
+            COMMENTS=$(gh api --paginate "repos/$REPO/issues/$PR_NUMBER/comments?per_page=100" --jq '.[].body' 2>/dev/null || true)
+            if ! grep -Fq "$MARKER" <<<"$COMMENTS"; then
+              gh pr comment "$PR_NUMBER" --repo "$REPO" --body "$MARKER
+Autonomous merge is blocked because this PR touches protected upstream/automation scope:
+
+\`\`\`
```

actionlint (1.7.12)

[error] could not parse as YAML: could not find expected ':'

(syntax-check)